### PR TITLE
RESTWS-836: Add docs for ConceptStateConversionResource2_0

### PR DESCRIPTION
**Pull request by BartlomiejRasztabiga**
Related issue: https://issues.openmrs.org/browse/RESTWS-836
Related PR: https://github.com/openmrs/openmrs-module-webservices.rest/pull/493

### CODE REVIEW

**Comment by dkayiwa** [comment]
Can you look at these merge conflicts?

**Comment by BartlomiejRasztabiga** [comment]
@dkayiwa Resolved

File: source/includes/Programs/state_conversions.md [hunk 1]
```diff
@@ -0,0 +1,362 @@
...
+    *[programWorkflow](#program-workflows)* | `ProgramWorkflow_UUID` | ProgramWorkflow resource UUID
+    *[programWorkflowState](#program-workflows)* | `ProgramWorkflowState_UUID` | ProgramWorkflowState resource UUID
+
+
+## Delete a Concept State Conversion
+
+> Delete a Concept State Conversion
+
+```shell
+DELETE /stateconversion/:uuid?purge=true
```
Comment (dkayiwa) [inline source/includes/Programs/state_conversions.md]: Did you forget to rename stateconversion?

Comment (BartlomiejRasztabiga) [reply source/includes/Programs/state_conversions.md]: I did :) Here's a fix: https://github.com/openmrs/openmrs-contrib-rest-api-docs/pull/158/commits/a0fc43c09f22bf697af442af2bb86ac9549c16ea

File: source/index.html.md [hunk 2]
```diff
@@ -53,6 +53,7 @@ includes:
   - Concepts/concept_stop_word
   - Programs/programs
   - Programs/program_workflows
+  - Programs/state_conversions
```
Comment (dkayiwa) [inline source/index.html.md]: concept_state_conversions?

Comment (BartlomiejRasztabiga) [reply source/index.html.md]: https://github.com/openmrs/openmrs-contrib-rest-api-docs/pull/158/commits/7e60019b9700a1619acde4f7b8886253c57aeea0

**Pull request merged**